app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,16 @@
 <resources>
     <string name="app_name">Weather</string>
+    <string name="title_ok">OK</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.title_ok` appears to be unused
 Unused resources make applications larger and slow down builds.